Tasks/ExtractFiles/task.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "5e1e3830-fbfb-11e5-aab1-090c92bc4988",
+  "name": "ExtractFiles",
+  "friendlyName": "Extract Files",
+  "description": "Extract files from a variety of archive types, e.g. .zip, .tar, .7z",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "Replace with markdown to show in help",
+  "category": "Utility",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [],
+  "version": {
+    "Major": "1",
+    "Minor": "0",
+    "Patch": "0"
+  },
+  "minimumAgentVersion": "1.98.1",
```

Comment:
Why are we demanding 1.98.1?  This means this task won't work for AA and other Dev14 QUs on prem servers.